profile.md

```diff
@@ -232,14 +233,45 @@ part of a Kubernetes API call:
   }
   ```
 
+## Bind Resource Object
+
+In the [Open Service Broker API specification](spec.md), requests to
+[create a Service Binding](spec.md#binding) can contain a `bind_resource`
+object in which Platforms MAY choose to add additional fields.
+
+### Cloud Foundry Bind Resource Object
+
+The following properties are defined for usage within a Cloud Foundry
+deployment:
+
+- `space_guid` (_optional_)  
```

Comment:
I think the use of `_optional_` here is new and I'd prefer to not confuse people so perhaps we should just say:
`This OPTIONAL property is the GUID of a ....`

Reply:
Done!